src/js/utils/timeout.js

```diff
@@ -0,0 +1,58 @@
+/* @flow */
+
+import Log, { init } from './debug';
+
+const _log: Log = init('Timeout', true);
+
+export default class Timeout {
```

Comment:
lets rename it to `InteractionTimeout` ? or `UITimeout`?

Reply:
Yep, I'll rename it to InteractionTimeout for more clarity.